analysis/diabetes_analysis.ipynb

```diff
@@ -0,0 +1,1247 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'logisticregression__C': 10}"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "best_params = random_search.best_params_ \n",
+    "best_params"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "b463a207-970a-4b87-b183-165a270216cf",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>mean_test_score</th>\n",
+       "      <th>mean_train_score</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>0.761717</td>\n",
+       "      <td>0.771416</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>0.761717</td>\n",
+       "      <td>0.771416</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>0.761717</td>\n",
+       "      <td>0.771416</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>0.759848</td>\n",
+       "      <td>0.770019</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>0.757961</td>\n",
+       "      <td>0.766764</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>0.743077</td>\n",
+       "      <td>0.749068</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>0.664798</td>\n",
+       "      <td>0.664804</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   mean_test_score  mean_train_score\n",
+       "4         0.761717          0.771416\n",
+       "5         0.761717          0.771416\n",
+       "6         0.761717          0.771416\n",
+       "3         0.759848          0.770019\n",
+       "2         0.757961          0.766764\n",
+       "1         0.743077          0.749068\n",
+       "0         0.664798          0.664804"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pd.DataFrame(random_search.cv_results_).sort_values(\n",
```

Comment:
The randomizedSearchCV results seem a bit low in terms of accuracy, should we consider alternative models or consider other scores such as f1? 

Understand f1 is content from 573 but seems a better indicator given class imbalance. 

If to consider alternative models, we can add a code chunk where we explore different models and pick the one with best cv score, like we did in lab in 571.